Modules/Sources/Storage/GRDB/Model/PersistedProductVariationImage.swift

```diff
@@ -34,7 +34,7 @@ extension PersistedProductVariationImage: FetchableRecord, PersistableRecord {
 
     public enum Columns {
         static let id = Column(CodingKeys.id)
-        static let productVariationID = Column(CodingKeys.productVariationID)
+        public static let productVariationID = Column(CodingKeys.productVariationID)
```

Comment:
We can make all these public but I'll do it in a future PR. 

I'm sure that periphery would moan about it, but actually it's part of the public interface for the persisted classes. Since the error messages are unclear, it's a better experience to make them public ahead of use.